packages/vue-components/README.md

```diff
@@ -1,86 +1,6 @@
 # MarkBind Vue components
 
-This folder contains MarkBind's [Vue.js](https://vuejs.org/) UI components.
+This package houses [Markbind's](https://markbind.org/) [Vue.js](https://vuejs.org/) UI components, used by the Markbind [core-web package](https://www.npmjs.com/package/@markbind/core-web).
```

Comment:
```suggestion
This package houses [MarkBind's](https://markbind.org/) [Vue.js](https://vuejs.org/) UI components, used by the MarkBind [core-web package](https://www.npmjs.com/package/@markbind/core-web).
```

Please note that the official name contains both capitalizations

Reply:
So sorry for the typo! I will make the relevant changes in the next commit!

Reply:
After searching for "Markbind" in the whole repo I realised that there are instances of "Markbind" in other files as well, should I make the relevant changes here or open an issue for this?

Reply:
Sure you can raise a new issue. I believe we have mostly converted everything, less those used in the coding context. Happy to double-check the instances you found.

Reply:
What are your thoughts on converting all instances of Markbind to MarkBind similar to what we did with frontmatter in code as well?

Reply:
Yes you can raise an issue(or direct PR) to highlight the changes needed.  If reasonable we should make the changes to keep it standardised, unless for circumstances such as naming convention in code or impacting existing exposed API. So probably need a review of the proposed changes to confirm. Previously some work was done for the same purpose https://github.com/MarkBind/markbind/pull/1828